storybook/stories/docs/inline-icon.js

```diff
@@ -2,11 +2,13 @@
  * External dependencies
  */
 import styled from '@emotion/styled';
-import { Icons } from '@storybook/components';
```

Comment:
`Icons` from `@storybook/components` is deprecated in favor of `@storybook/icons`, see https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#icons-is-deprecated